middleware/session/store.go

```diff
@@ -79,7 +79,11 @@ func (s *Store) Get(c *fiber.Ctx) (*Session, error) {
 		} else if err != nil {
 			return nil, err
 		} else {
+			// raw is nil, which means id is not in the storage
+			// so it means that id is not valid (mainly because of id is expired or user provides an invalid id)
+			// therefore, we regenerate a id
 			sess.fresh = true
```

Comment:
can you create a method that combines the two commands and use it in sess.Regenerate and store, think that would be even cleaner

```go
func (s *Session) refresh() {
    s.id = s.config.KeyGenerator()
    s.fresh = true
}
```

Reply:
Done.